website/source/docs/builders/vmware-iso.html.md

```diff
@@ -28,6 +28,25 @@ scratch, booting it, installing an OS, provisioning software within the OS, then
 shutting it down. The result of the VMware builder is a directory containing all
 the files necessary to run the virtual machine.
 
+### VMware Player on Linux
+
+There are (as of 2016-Apr-18) two versions of VMware Player currently available for download:
+
+* [7.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_player/7_0)
+* [12.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_workstation_player/12_0)
```

Comment:
I think the vSphere issue is related to the compatibility version, which can be specified in VMX. You're right that this is more complicated, though.

If you think it's substantially easier to write the instructions for Player 7 I'm OK with using that instead. Since both are free anyway I don't think it matters too much.